test/powershell/Modules/Microsoft.PowerShell.Core/Enter-PSHostProcess.Tests.ps1

```diff
@@ -132,6 +132,9 @@ Describe "Enter-PSHostProcess tests" -Tag Feature {
                 $npInfo = [System.Management.Automation.Runspaces.NamedPipeConnectionInfo]::new($pwshId)
```

Comment:
@TylerLeonhardt, your last commit had 1 failures in `PowerShell-CI-linux`
Enter-PSHostProcess tests.By Process Id.Can enter using NamedPipeConnectionInfo
```powershell
You cannot call a method on a null-valued expression.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Core/Enter-PSHostProcess.Tests.ps1: line 159
159:                 $ps.Dispose()

```

Reply:
@TylerLeonhardt  The $ps and $rs variables should be check for null, in case an exception is thrown before they are assigned.

Reply:
added `?` check